index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html title="Project_1" lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="//db.onlinewebfonts.com/c/809e4b078034095ff3d1fb1097aadcb4?family=Old+English+(Let)" rel="stylesheet" type="text/css"/> 
+    <script src="https://kit.fontawesome.com/cf2fdf395d.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav class="navbar">
+        <i class=" nav nav1 fas fa-bars"></i>
+        <i class="nav nav2 fas fa-search"></i>
+        <a class="nav nav3" href="#home">Space & Cosmos</a>
+        <a class="nav nav4 cursivetitle" href="#home" >New York Times</a>
+        <button class="nav btn" type="submit">Subscribe</button>
+        <button class="nav btn" type="submit">Sign up</button>
+    </nav>
+    <div class="containerparent">
+        <div class="middle-section header">
+            <h1>Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+        </div>    
+        <div class="img1">
+            <img class="image1" src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos/18cosmos-superJumbo.jpg?quality=90&auto=webp" alt="Cosmos">
+            <p>Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago</p>
+            <p>Rick Friedman for The New York Times</p>
+       </div>
+       <div>
+            <h2>By <a href="#top">Dennis Overbye</a></h2>   
+            <h3>March 17, 2014</h3>
+            <p>Sign up for Science Times: Get stories that capture the wonders of nature, the cosmos and the human body.</p>
+           <button class="btn" type="submit">Sign Up</button>
+       </div>
+        <div class="middle-section">
```

Comment:
## Improve the main artcile seciion and nabvabr.
- [ ] Take note of the colour topography of the buttons and ensure they are of the same visual appeal as that of the source page
- [ ] Improve the logo link. You will need to replace it with an image logo link instead.
- [ ] Add the missing social links atop the main image.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/mariordgez/Project_1_HTML/project-1-microverse/index.html)|
|:--:|:--:|
|![source](https://user-images.githubusercontent.com/22345961/112551597-51cd7500-8dd2-11eb-91e2-0c1e16398378.png)|![clone](https://user-images.githubusercontent.com/22345961/112551620-585bec80-8dd2-11eb-811c-bef44674c3b8.png)|